mediagalaxy.ro,"HTTPSConnectionPool(host='mediagalaxy.ro', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='mediagalaxy.ro', port=80): Read timed out. (read timeout=2)"
mediclinic.ae,"HTTPSConnectionPool(host='mediclinic.ae', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.mediclinic.ae', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
medicosdeelsalvador.com,"HTTPSConnectionPool(host='medicosdeelsalvador.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='medicosdeelsalvador.com', port=80): Read timed out. (read timeout=2)"
mediji.net,"HTTPSConnectionPool(host='mediji.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mediji.net', port=443): Read timed out. (read timeout=2)"
medinet.gov.vn,"HTTPSConnectionPool(host='medinet.gov.vn', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f5ff7ceec10>, 'Connection to medinet.gov.vn timed out. (connect timeout=2)'))","HTTPConnectionPool(host='medinet.gov.vn', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f5ff6af17c0>, 'Connection to medinet.gov.vn timed out. (connect timeout=2)'))"
medvestnik.ru,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
medwave.cl,"HTTPSConnectionPool(host='medwave.cl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff84ab160>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='medwave.cl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff5065340>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mefos.hr,"HTTPSConnectionPool(host='mefos.hr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff85c3490>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mefos.hr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff530d3d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
megashop.hr,"HTTPSConnectionPool(host='www.megashop.hr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.megashop.hr', port=443): Read timed out. (read timeout=2)"
megashop.si,"HTTPSConnectionPool(host='www.megashop.si', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.megashop.si', port=443): Read timed out. (read timeout=2)"
meiji.ac.jp,"HTTPSConnectionPool(host='meiji.ac.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff57db670>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='meiji.ac.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff3636490>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
melody4arab.xyz,"HTTPSConnectionPool(host='melody4arab.xyz', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff2224f70>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='melody4arab.xyz', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff6396130>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
men.gov.ma,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
mendoza.gov.ar,"HTTPSConnectionPool(host='mendoza.gov.ar', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff67b9f10>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mendoza.gov.ar', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff6513fa0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
menlhk.go.id,"HTTPSConnectionPool(host='menlhk.go.id', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff3580fd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='menlhk.go.id', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff22329d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
menofia.edu.eg,"HTTPSConnectionPool(host='menofia.edu.eg', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff25f59a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='menofia.edu.eg', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff66a5280>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
menoinfo.fi,"HTTPSConnectionPool(host='menoinfo.fi', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff3370c10>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='menoinfo.fi', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff2aee5e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
menupages.ae,"HTTPSConnectionPool(host='menupages.ae', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='menupages.ae', port=443): Read timed out. (read timeout=2)"
menupages.ie,"HTTPSConnectionPool(host='menupages.ie', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f5ff6cfcca0>, 'Connection to menupages.ie timed out. (connect timeout=2)'))","HTTPConnectionPool(host='menupages.ie', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f5ff804f5e0>, 'Connection to menupages.ie timed out. (connect timeout=2)'))"
mep.go.cr,Exceeded 30 redirects.,Exceeded 30 redirects.
mercadopublico.cl,"HTTPSConnectionPool(host='mercadopublico.cl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5fef805970>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mercadopublico.cl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff7140b50>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mercedes.gob.ar,"HTTPSConnectionPool(host='mercedes.gob.ar', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff5495be0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mercedes.gob.ar', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff6743e20>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mercubuana-yogya.ac.id,"HTTPSConnectionPool(host='mercubuana-yogya.ac.id', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='mercubuana-yogya.ac.id', port=80): Read timed out. (read timeout=2)"
meteo.it,"HTTPSConnectionPool(host='meteo.it', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))","HTTPSConnectionPool(host='meteo.it', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))"
meti.go.jp,"HTTPSConnectionPool(host='meti.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff5aeab20>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='meti.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff80669a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
metjekutindevut.nl,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
metro.tokyo.jp,"HTTPSConnectionPool(host='metro.tokyo.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff7314fa0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='metro.tokyo.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff68eab50>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
metrouniv.ac.id,"HTTPSConnectionPool(host='metrouniv.ac.id', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='metrouniv.ac.id', port=443): Read timed out. (read timeout=2)"
metsalehti.fi,"HTTPSConnectionPool(host='metsalehti.fi', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff347ecd0>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPSConnectionPool(host='www.metsalehti.fi', port=443): Read timed out. (read timeout=2)"
mext.go.jp,"HTTPSConnectionPool(host='mext.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff2224730>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mext.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff1149a90>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mfa.ee,"HTTPSConnectionPool(host='mfa.ee', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff6d1f430>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mfa.ee', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5fef04e460>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mfa.gov.sg,"HTTPSConnectionPool(host='mfa.gov.sg', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'mfa.gov.sg' doesn't match 'www.mfa.gov.sg'"")))","HTTPSConnectionPool(host='mfa.gov.sg', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'mfa.gov.sg' doesn't match 'www.mfa.gov.sg'"")))"
mfa.gov.tr,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","HTTPSConnectionPool(host='mfa.gov.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
mfe.govt.nz,"HTTPSConnectionPool(host='mfe.govt.nz', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='mfe.govt.nz', port=80): Read timed out. (read timeout=2)"
mhlw.go.jp,"HTTPSConnectionPool(host='mhlw.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff34c52e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mhlw.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5fef2ff940>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
michoacan.gob.mx,"HTTPSConnectionPool(host='michoacan.gob.mx', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='michoacan.gob.mx', port=443): Read timed out. (read timeout=2)"
mie.lg.jp,"HTTPSConnectionPool(host='mie.lg.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff5905850>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mie.lg.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff51be610>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
migdalor-news.co.il,"HTTPSConnectionPool(host='www.migdalor-news.co.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.migdalor-news.co.il', port=443): Read timed out. (read timeout=2)"
mikmak.co.il,"HTTPSConnectionPool(host='mikmak.co.il', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f5fee954040>, 'Connection to mikmak.co.il timed out. (connect timeout=2)'))","HTTPConnectionPool(host='mikmak.co.il', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f5ff520c430>, 'Connection to mikmak.co.il timed out. (connect timeout=2)'))"
mikronis.hr,"HTTPSConnectionPool(host='mikronis.hr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.mikronis.hr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
milejemplos.com,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
min-mave.dk,"HTTPSConnectionPool(host='min-mave.dk', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='min-mave.dk', port=443): Read timed out. (read timeout=2)"
min-saude.pt,"HTTPSConnectionPool(host='min-saude.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff66f2ca0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='min-saude.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff3b8e940>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
minciencias.gov.co,"HTTPSConnectionPool(host='minciencias.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='minciencias.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
mindop.sk,"HTTPSConnectionPool(host='mindop.sk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPConnectionPool(host='mindop.sk', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f5ff5709d90>, 'Connection to mindop.sk timed out. (connect timeout=2)'))"
minec.gob.sv,"HTTPSConnectionPool(host='minec.gob.sv', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f5fee5835b0>, 'Connection to minec.gob.sv timed out. (connect timeout=2)'))","HTTPConnectionPool(host='minec.gob.sv', port=80): Read timed out. (read timeout=2)"
minec.gob.ve,"HTTPSConnectionPool(host='minec.gob.ve', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5fef04edf0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='minec.gob.ve', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff7372fa0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mineco.gob.gt,"HTTPSConnectionPool(host='mineco.gob.gt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff56a7af0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mineco.gob.gt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff71f0430>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mined.gob.sv,"HTTPSConnectionPool(host='mined.gob.sv', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff550f760>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mined.gob.sv', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff8604ee0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
minedu.gob.bo,"HTTPSConnectionPool(host='minedu.gob.bo', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))","HTTPSConnectionPool(host='minedu.gob.bo', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))"
minedu.sk,"HTTPSConnectionPool(host='minedu.sk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff56caac0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='minedu.sk', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5feed2da60>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mineducacion.gov.co,"HTTPSConnectionPool(host='mineducacion.gov.co', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff0adce80>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='mineducacion.gov.co', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff7e20ee0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
minetilbud.dk,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
minfin.bg,"HTTPSConnectionPool(host='www.minfin.bg', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.minfin.bg', port=443): Read timed out. (read timeout=2)"
mininterior.gov.co,"HTTPSConnectionPool(host='mininterior.gov.co', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f5ff69eb640>, 'Connection to mininterior.gov.co timed out. (connect timeout=2)'))","HTTPConnectionPool(host='mininterior.gov.co', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f5fee726100>, 'Connection to mininterior.gov.co timed out. (connect timeout=2)'))"
ministeriodegobierno.gob.ec,"HTTPSConnectionPool(host='ministeriodegobierno.gob.ec', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'ministeriodegobierno.gob.ec' doesn't match either of 'www.agricultura.gob.ec', 'agricultura.gob.ec'"")))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
ministeriodesalud.go.cr,"HTTPSConnectionPool(host='ministeriodesalud.go.cr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5feeee2d90>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ministeriodesalud.go.cr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff1444700>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
minnanokaigo.com,"HTTPSConnectionPool(host='www.minnanokaigo.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.minnanokaigo.com', port=443): Read timed out. (read timeout=2)"
minsa.gob.pe,"HTTPSConnectionPool(host='minsa.gob.pe', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff5814340>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='minsa.gob.pe', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff2355d60>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
minsalud.gov.co,"HTTPSConnectionPool(host='minsalud.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.minsalud.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
mintrabajo.gov.co,"HTTPSConnectionPool(host='mintrabajo.gov.co', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff556ff40>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mintrabajo.gov.co', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff6d870d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mintransporte.gov.co,"HTTPSConnectionPool(host='mintransporte.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='mintransporte.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
minvu.cl,"HTTPSConnectionPool(host='minvu.cl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
minyu-net.com,"HTTPSConnectionPool(host='minyu-net.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))","HTTPSConnectionPool(host='minyu-net.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))"
mise.gov.it,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
missbloom.bg,"HTTPSConnectionPool(host='missbloom.bg', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f5ff259fa00>, 'Connection to missbloom.bg timed out. (connect timeout=2)'))","HTTPConnectionPool(host='missbloom.bg', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f5ff6d87eb0>, 'Connection to missbloom.bg timed out. (connect timeout=2)'))"
missmundo.nl,"HTTPSConnectionPool(host='missmundo.nl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff4c3f310>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='missmundo.nl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff3661610>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
misumi-ec.com,"HTTPSConnectionPool(host='misumi-ec.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff787bc40>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='misumi-ec.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff73386a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mizuhobank.co.jp,"HTTPSConnectionPool(host='mizuhobank.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff7346280>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mizuhobank.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5fee865910>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mj.pt,"HTTPSConnectionPool(host='mj.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff7f506a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mj.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5feef11970>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mju.ac.th,"HTTPSConnectionPool(host='mju.ac.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff5aa6580>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mju.ac.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff7fece80>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mku.edu.tr,"HTTPSConnectionPool(host='mku.edu.tr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff7c6a4c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mku.edu.tr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5fef505610>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mladinska.com,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
mlit.go.jp,"HTTPSConnectionPool(host='mlit.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff7956460>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mlit.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff8369040>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mlsi.gov.cy,"HTTPSConnectionPool(host='mlsi.gov.cy', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff61060a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mlsi.gov.cy', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff5284b20>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mm.lv,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
mobilnisvet.com,"HTTPSConnectionPool(host='mobilnisvet.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='mobilnisvet.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
moda.sk,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
moe.edu.sg,"HTTPSConnectionPool(host='moe.edu.sg', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff65aa790>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='moe.edu.sg', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff7ae0dc0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
moe.go.th,"HTTPSConnectionPool(host='moe.go.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff372a9d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='moe.go.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff6889b50>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
moe.gov.jo,"HTTPSConnectionPool(host='moe.gov.jo', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='moe.gov.jo', port=443): Read timed out. (read timeout=2)"
moe.gov.lk,"HTTPSConnectionPool(host='moe.gov.lk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='moe.gov.lk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
moe.gov.my,"HTTPSConnectionPool(host='moe.gov.my', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: TLSV1_UNRECOGNIZED_NAME] tlsv1 unrecognized name (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.moe.gov.my', port=443): Read timed out. (read timeout=2)"
mof.gov.vn,"HTTPConnectionPool(host='mof.gov.vn', port=80): Max retries exceeded with url: /webcenter/portal/btc?_afrLoop=1621525455.989 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5fedfa9850>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPSConnectionPool(host='mof.gov.vn', port=443): Read timed out. (read timeout=2)"
mofa.go.jp,"HTTPSConnectionPool(host='mofa.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff62f04c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mofa.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff7648f10>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mofa.gov.bh,"HTTPSConnectionPool(host='mofa.gov.bh', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mofa.gov.bh', port=443): Read timed out. (read timeout=2)"
moh.gov.bh,"HTTPSConnectionPool(host='moh.gov.bh', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ff582f940>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='moh.gov.bh', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff5e47a90>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
moh.gov.jo,"HTTPSConnectionPool(host='moh.gov.jo', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='moh.gov.jo', port=80): Read timed out. (read timeout=2)"
moh.gov.my,"HTTPSConnectionPool(host='moh.gov.my', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'moh.gov.my' doesn't match '*.moh.gov.my'"")))","HTTPSConnectionPool(host='moh.gov.my', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'moh.gov.my' doesn't match '*.moh.gov.my'"")))"
mohap.gov.ae,"HTTPSConnectionPool(host='mohap.gov.ae', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: TLSV1_UNRECOGNIZED_NAME] tlsv1 unrecognized name (_ssl.c:1124)')))","HTTPConnectionPool(host='www.mohap.gov.ae', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5ff7750b50>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mohre.gov.ae,"HTTPSConnectionPool(host='mohre.gov.ae', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.mohre.gov.ae', port=443): Read timed out. (read timeout=2)"
